README.md

```diff
@@ -2,99 +2,41 @@
 [![codecov](https://codecov.io/gh/pendulum-project/ntpd-rs/branch/main/graph/badge.svg?token=WES1JIYUJH)](https://codecov.io/gh/pendulum-project/ntpd-rs)
 [![Crates.io](https://img.shields.io/crates/v/ntpd.svg)](https://crates.io/crates/ntpd)
 
-
 # ntpd-rs
 
-ntpd-rs is an NTP implementation written in Rust, with a focus on security and
-stability. It includes client and server functionality and supports NTS.
+nptd-rs is a tool for synchronizing your computer's clock. Implementing the NTP and NTS protocols, it is written in Rust, with a focus on security and stability. It includes both client and server support.
 
 If a feature you need is missing please let us know by opening an issue.
 
```

Comment:
At first at skipped over the documentation link below, I think the documentation website deserves a more prominent location in the README to direct more people to the getting started guides sooner.

```suggestion

## Documentation

Be sure to check out the [documentation website] as it includes guides on getting started, installation and migration, as well as a high-level overview of the code structure.

```

Reply:
maybe this can also be one of those badge thingies?